### PR TITLE
Run yamls additions

### PR DESCRIPTION
**Pull request by liranszlak**
This adds some additional default yamls and parameters for the user to set more easily.

### CODE REVIEW

File: run/rda_pre_train_config.yaml [hunk 1]
```diff
@@ -24,10 +24,15 @@ data_module:
 trainer:
   learning_rate: 1.0e-5
   losses:
-    - name: cross_entropy
-      label_column_name: ${label_column_name}
+    - field_name: genes
+      name: cross_entropy
+    - field_name: expressions
+      name: mse
+      ignore_zeros: true
```
Comment (mmdanziger) [inline run/rda_pre_train_config.yaml:31]:
```suggestion
      ignore_zero: true
```

very important

File: run/transform_and_fine_tune_config.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+data_module:
+  max_length: 256
+  data_dir: /PATH/TO/DATA/
+  processed_name: processed_data
+  limit_genes: protein_coding
+  transform_datasets: true
+
+  transform_kwargs:
+    source_h5ad_file_name: "/dccstor/bmfm-targets/data/omics/transcriptome/epilepsy/brain/epilepsy_all_brain.h5ad"
```
Comment (mmdanziger) [inline run/transform_and_fine_tune_config.yaml:25]: can we handle this better? there are two paths to data... and one is non-generic

File: run/transform_and_fine_tune_config.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    transforms:
+       - transform_name: KeepVocabGenesTransform
+         transform_args:
+           vocab_identifier: all_genes
+
+       - transform_name: TPMNormalizationTransform
+         transform_args:
+           gtf_file: /dccstor/bmfm-targets/data/omics/transcriptome/bulkRNA/GRCh38_latest_genomic.gtf
```
Comment (mmdanziger) [inline run/transform_and_fine_tune_config.yaml:33]: maybe pop this out as an environment variable for portability?

**Review by mmdanziger — CHANGES_REQUESTED** [review]
one typo that is very important, also please install pre-commit locally. follow the instructions inside the precommitconfig yaml

File: run/transform_and_fine_tune_config.yaml (hunk 2 above)
Comment (liranszlak) [reply run/transform_and_fine_tune_config.yaml:25]: oh sorry that was a miss on my end, I forgot to change it to a generic path

File: run/README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Script for running bmfm-targets-scbert
 
 ```bash
-bmfm-targets-scbert -cn fine_tune_train_and_test.yaml -cd run
+bmfm-targets-scbert -cn prediction_train_and_test.yaml -cd run
```
Comment (mmdanziger) [inline run/README.md:4]:
```suggestion
bmfm-targets-scbert -cn prediction_train_and_test -cd run
```

File: run/data_module/base_seq_cls.yaml [hunk 5]
```diff
@@ -7,4 +7,5 @@ max_length: 512
 limit_dataset_samples: null
 shuffle: true
 sequence_order: "random"
+transform_datasets: false
```
Comment (mmdanziger) [inline run/data_module/base_seq_cls.yaml:10]: i think this setting should be only in the final config, not the other levels
so when you are doing your global setting you decide whether to retransform or not. if it is repeated it gets confusing
```suggestion
```

File: run/data_module/base_language_modeling.yaml [hunk 6]
```diff
@@ -7,4 +7,5 @@ max_length: 512
 limit_dataset_samples: null
 shuffle: true
 sequence_order: "random"
+transform_datasets: false
```
Comment (mmdanziger) [inline run/data_module/base_language_modeling.yaml:10]:
```suggestion
```

**Review by mmdanziger — APPROVED** [review]
thanks @liranszlak minor comments inside no need for rereview
